### PR TITLE
reprex v1.0.0

### PR DESCRIPTION
**Pull request by jennybc**

### CODE REVIEW

File: content/blog/reprex-1-0-0/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+install.packages("reprex")
+```
+
+It turns out we've never blogged about reprex here on [tidyverse.org](https://www.tidyverse.org), so we start with a general overview for newcomers, then close with a summary of recent changes of interest to existing users.
+
+You can see a full list of changes in the [release notes](https://reprex.tidyverse.org/news/index.html#reprex-1-0-0-2021-01-27).
+
+## Why reprex exists
+
+reprex is a convenience package that combines the power of R Markdown with conventional wisdom about what makes a good reproducible example.
```
Comment (hadley) [inline content/blog/reprex-1-0-0/index.Rmd:39]:
```suggestion
reprex is a convenience package that combines the power of RMarkdown with conventional wisdom about what makes a good reproducible example.
```

File: content/blog/reprex-1-0-0/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    Levels: Hello, 
+    > y
+    [1] world!
+    Levels: world!
+    > c(x, y)
+    [1] 1 1
+    ```
+
+    You need to remove the `>` prompts and delete all output.
+    And don't get me started on the problem of screenshots.
```
Comment (hadley) [inline content/blog/reprex-1-0-0/index.Rmd:81]: Let alone the problem of photos of screens!

Comment (hadley) [reply content/blog/reprex-1-0-0/index.Rmd:81]: Oh I see you mention that below

File: content/blog/reprex-1-0-0/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+Who knew?
+`r emo::ji("shrug")`
+This is also a good time to remind everyone that `venue = "r"` or `reprex_r()` are great ways to create larger Slack code snippets.
+Before you finish by clicking "Create snippet", select `R` from the "Type" dropdown to get nice syntax highlighting.
+
+[^3]: You can disable the WYSIWYG Slack message interface in **Preferences \> Advanced**.
+    Select the **Format messages with markup setting**.
+    The [Slack section of The Markdown Guide](https://www.markdownguide.org/tools/slack/) is helpful for figuring out which subsets of Markdown are supported in different parts of Slack.
+
+Stack Overflow now supports fenced code blocks, which means that the `"so"` venue is no longer necessary.
```
Comment (hadley) [inline content/blog/reprex-1-0-0/index.Rmd:147]: Maybe not worth mentioning in the blog post?

**Review by hadley — APPROVED** [review]

File: content/blog/reprex-1-0-0/index.Rmd (hunk 1 above)
Comment (jennybc) [reply content/blog/reprex-1-0-0/index.Rmd:39]: I was following what I see here: https://rmarkdown.rstudio.com. I did contemplate space vs no space.

File: content/blog/reprex-1-0-0/index.Rmd (hunk 2 above)
Comment (jennybc) [reply content/blog/reprex-1-0-0/index.Rmd:81]: Saving the best for last! 📸

**Pull request merged**